simapp/simd/cmd/root.go

```diff
@@ -25,6 +25,7 @@ import (
 	"github.com/cosmos/cosmos-sdk/client/rpc"
 	"github.com/cosmos/cosmos-sdk/codec"
 	"github.com/cosmos/cosmos-sdk/server"
+	tmcmd "github.com/cosmos/cosmos-sdk/server"
```

Comment:
```suggestion
```

This package is already imported. You can do `server.QueryBlocksCmd()` instead.

Reply:
gotcha thanks